tests/post/steps/test_logs.py

```diff
@@ -15,12 +15,14 @@ def check_logs(host):
             '--no-headers -o custom-columns=":metadata.name"'
         )
         for pod_id in pods_list.split('\n'):
+            if 'salt-master' in pod_id:
```

Comment:
Why do we skip `salt-master`?

Maybe add a comment.

Reply:
I guess sth wrong here, this changes made by me long ago...
and we skip salt-master because its logs is in `info` mode so you dont have logs usually

Reply:
So, originally that was indeed the reason @sayf-eddine-scality 

There's no another reason: we run 2 containers in the `salt-master` Pod, which makes `kubectl logs ...` bail out, unless we pass in a container name.

Now, given there's been no interactions with SaltAPI at this point, the logs may again be 'empty' (similar to the original issue), so keeping this as-is. However, I'll add a comment explaining the two reasons (old one and new one).